src/formatDate.js

```diff
@@ -8,7 +8,34 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const oldSeparator = fromFormat.slice(3);
+  const newSeparator = toFormat.slice(3);
+  const splitDate = date.split(oldSeparator);
+
+  const newDate = [];
+  const objectDate = {};
+
+  for (let i = 0; i < splitDate.length; i++) {
+    objectDate[fromFormat[i]] = splitDate[i];
+  }
+
+  if (fromFormat.includes('YY') && toFormat.includes('YYYY')) {
```

Comment:
The condition to check if the `fromFormat` includes 'YY' and `toFormat` includes 'YYYY' is correct, but it is not necessary to use the `includes` method twice. You can store the results of `fromFormat.indexOf('YY')` and `toFormat.indexOf('YYYY')` in variables and then check if those indexes are greater than or equal to 0. This will avoid searching the arrays multiple times.